app/routes/owner/repositories.js

```diff
@@ -1,10 +1,9 @@
 import TravisRoute from 'travis/routes/basic';
 import config from 'travis/config/environment';
-import { service } from 'ember-decorators/service';
-import { computed } from 'ember-decorators/object';
+import { inject as service } from '@ember/service';
 
 export default TravisRoute.extend({
-  @service auth: null,
+  features: service(),
```

Comment:
It would match the rest of the codebase more to use `@service features: null` instead here, I think, did that not work for some reason?

Reply:
Ah, right, thanks for pointing 👍 that's a habit :)